docarray/array/storage/qdrant/backend.py

```diff
@@ -175,10 +176,11 @@ def _get_offset2ids_meta(self) -> List[str]:
         ).result.payload['offset2id']
 
     def _update_offset2ids_meta(self):
+        vectors_config = VectorParams(size=1, distance=Distance.COSINE)
```

Comment:
same here

Reply:
not the same, this is just a hack for offset2id

Reply:
@dongxiang123  no need to specify n_dim for offset2id collection.
Please re-apply your old changes in `_update_offset2ids_meta` method:
`vectors_config = VectorParams(size=1, distance=Distance.COSINE)`